network.bisq.Bisq.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Name=Bisq
+Comment=A decentralized bitcoin exchange network.
+Exec=/app/bin/Bisq
```

```suggestion
Exec=Bisq
```

it's never the full path